### PR TITLE
A proposal on how styling could (should?) be made.

### PR DESCRIPTION
**Pull request by dumbNickname**
Style component with css, not html attributes (where not needed), improved mobile view, removed deprecated css, moved data to .ts file to keep view definition not that tightly coupled with data.

### CODE REVIEW

**Comment by sjimenez77** [comment]
Thanks Bartek! I will try to buy some time and review your PR ASAP, as I am on holidays until Wednesday.

**Comment by dumbNickname** [comment]
No problem. I was hoping to catch you today to agree on some other things, but enjoy your holiday! I will try to pick a way on my own and then let you decide if it was implemented as you would want it :P

File: angular/src/app/home/home.component.scss [hunk 1]
```diff
@@ -33,34 +18,58 @@
...
+  .home-card {
+    margin-top: -75px;
+    margin-left: 48px;
+    margin-right: 48px;
+    padding: 16px;
+
+    .tiles {
+      flex-direction: row;
+
+      tile-content {
```
Comment (sjimenez77) [inline angular/src/app/home/home.component.scss]: I guess that this should be `.tile-content` right?

Comment (dumbNickname) [reply angular/src/app/home/home.component.scss]: yes :D obviously it should :P

**Pull request merged**